ote/src/clj/ote/services/admin.clj

```diff
@@ -73,6 +76,36 @@
          search-params
          {:specql.core/order-by ::t-service/operator-name})))
 
+(defn distinct-by [f coll]
+  (let [groups (group-by f coll)]
+    (map #(first (groups %)) (distinct (map f coll)))))
+
+(defn- business-id-report [db user query]
+  (let [services (if
+                   (or
+                     (nil? (:business-id-filter query))
+                     (= :ALL (:business-id-filter query))
+                     (= :services (:business-id-filter query)))
+                   (fetch-service-business-ids db)
+                   nil)
+        operators (if
+                   (or
+                     (nil? (:business-id-filter query))
+                     (= :ALL (:business-id-filter query))
+                     (= :operators (:business-id-filter query)))
+                    (fetch-operator-business-ids db)
+                   nil)
+
+        report (cond
```

Comment:
Why is this  `cond` structure necessary? just `(concat services operators)`

Reply:
Removed cond and replaced it with concat